spec/features/view_message_spec.rb

```diff
@@ -0,0 +1,9 @@
+feature 'being able to see messages' do 
```

Comment:
I'm not sure this test is necessary? It feels like you've covered it in add_peep_spec.rb